clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -515,5 +515,35 @@ def BitfieldInfoAttr : CIR_Attr<"BitfieldInfo", "bitfield_info"> {
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// AddressPointAttr
+//===----------------------------------------------------------------------===//
+
+def AddressPointAttr : CIR_Attr<"AddressPoint", "address_point"> {
```

Comment:
Is there any reason to have this as a special attribute and not just two `I32Attr` in `VTableAddrPointOp`? 

I scanned through incubator codebase and all uses I found it is always accessed directly to index or offset, e.g. `op.getAddressPointAttr().getIndex()`, `op.getAddressPointAttr().getOffset()`.

Therefore I believe we can remove the attribute entirely and just have `op.getAddressPointIndex()` and `op.gettAddressPointOffset()`? 

This can clean up address point op assembly as weel to something like:
```
cir.vtable_address_point @_ZTV1B [index = 0, offset = 2] : !cir.ptr<!cir.ptr<() -> i32>>
```

^ This just my suggestion removing also some parenthesis and adding `_` after `vtable` in op name.

Reply:
> Do we have any consensus on how this should be handled?

We probably agree it should be easy to read / informative for passes?

> I've left the attribute, based on supposition of future usefulness, but I'm not opposed to separate attributes as @xlauko suggests.

SGTM - I'm not particularly tied to whether this is encoded directly in the op with pretty printing (e.g. `index = ...`, etc) or via an attribute wrapper.